### ISSUE TITLE
Unstable training

### ISSUE THREAD

**Issue by okbalefthanded** [yaoli90/Conv-CA#2]
Hi, first of all thanks for the clear and concise code.
While attempting the reproduce the results, I've noticed an unstable training behavior for time window length 0.2 seconds (50 samples), the loss goes to Nan after approximately 50 epochs. I had to run multiple execution to reach a successful training. this behavior was seen only in tw of 0.2 seconds.

btw, I'm using google colab to run experiments with TF 2.5.0, Python version 3.7. and NVIDIA Tesla T4 as GPU.

**Comment by yaoli90** [comment]
I received another similar report saying the training is unstable and goes to NaN at around 50 epochs. He has set gradient clipping at 5. He was also using google platform running the code. After he switched to a new environment, the code works.

I have tested the code recently in a Ubuntu environment,
My library versions:
Keras 2.3.1
Tensor flow 1.14.0

Hope it helps.

**Comment by okbalefthanded** [comment]
Thanks for the clarification, I'll try with an earlier TF version.
